presto-main/src/main/java/com/facebook/presto/execution/scheduler/nodeSelection/SimpleNodeSelector.java

```diff
@@ -98,6 +102,31 @@ public SimpleNodeSelector(
         this.maxTasksPerStage = maxTasksPerStage;
     }
 
+    public int getMinCandidates()
+    {
+        return minCandidates;
+    }
+
+    public boolean isIncludeCoordinator()
+    {
+        return includeCoordinator;
+    }
+
+    public int getMaxSplitsPerNode()
+    {
+        return maxSplitsPerNode;
+    }
+
+    public int getMaxPendingSplitsPerTask()
+    {
+        return maxPendingSplitsPerTask;
+    }
+
+    public int getMaxTasksPerStage()
+    {
+        return maxTasksPerStage;
+    }
```

Comment:
I would recommend avoiding public methods for these, and simply injecting the values where needed in `SimpleTTLNodeSelector`.